src/test/java/de/novatec/domainprimitives/object/testdata/PersonId.java

```diff
@@ -0,0 +1,11 @@
+package de.novatec.domainprimitives.object.testdata;
+
+import de.novatec.domainprimitives.type.PrimitiveType;
+
+import static de.novatec.domainprimitives.validation.Constraints.isGreatThanOrEqual;
+
+public class PersonId extends PrimitiveType<Long> {
+    public PersonId(Long value) {
+        super(value, "PersonId", isGreatThanOrEqual(1L));
```

Comment:
Do wie need the label for demonstration purpose?

Reply:
No, I will remove it in all test data objects.